packages/guide-cue/src/GuideCue.tsx

```diff
@@ -117,6 +118,14 @@ function GuideCue({
     ...tooltipProps,
   };
 
+  const [shouldRender, setShouldRender] = useState(false);
+
+  useIsomorphicLayoutEffect(() => setShouldRender(true), []);
+
+  if (!shouldRender) {
+    return null;
+  }
+
```

Comment:
Do the tests fail without this? Just wondering if this is really needed

Reply:
Without this I get a window is not defined error coming from the popover component from the guidecue test. Popover is also using `useIsomorphicLayoutEffect` so i'm not 100% sure why that error is showing up.